migration/migrations/20200323014820-create-plan.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    return queryInterface.createTable('Orders', {
```

Comment:
@arrudaricardo here the right it would be Plans?
```suggestion
    return queryInterface.createTable('Plans', {
```